schema/meter_provider.json

```diff
@@ -2,7 +2,7 @@
     "$id": "https://opentelemetry.io/otelconfig/meter_provider.json",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
     "title": "MeterProvider",
-    "type": ["object", "null"],
+    "type": "object",
```

Comment:
Setting `meter_provider:` is not a sensible thing to do. If you want a noop meter_provider, omit `meter_provider` altogether.